### DISCUSSION TITLE
can i move the words "powered by mediacms" to the "About" page?

### DISCUSSION THREAD

**Discussion by egiaggata** [mediacms-io/mediacms#945] [Q&A]
can i move the words "powered by mediacms" to the "About" page?

Thanks

**Comment by QuestOAuth74** [comment]
Yes you can. If you are on the command line. sudo nano /home/mediacms.io/mediacms/templates/config/installation/contents.html 

edit this line> 'Powered by <a href="//mediacms.io" title="mediacms.io" target="_blank">mediacms.i>

All the best.
